config/schemas/mod.json

```diff
@@ -122,6 +122,11 @@
 			"description" : "List of mods that are required to run this one",
 			"items" : { "type" : "string" }
 		},
+		"softDepends" : {
+			"type" : "array",
+			"description" : "List of mods if exist, it should be loaded before this one",
```

Comment:
If I understood correctly, the soft dependency must be manualy enabled?

"List of mods where each mod, only if enabled, must be loaded ahead of this mod"

Reply:
> If I understood correctly, the soft dependency must be manualy enabled?
> 
> "List of mods where each mod, only if enabled, must be loaded ahead of this mod"

From what I understand, the soft dependency allows to overwrite some mods if they are enabled.

Reply:
I add more texts to explain it.